vite.config.js

```diff
@@ -39,6 +41,18 @@ export default defineConfig(() => {
       },
       extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
     },
+    build: {
+      rollupOptions: {
+        external: [
+          'vuetify/lib/components/VBtn/index.mjs',
```

Comment:
Was getting some build errors with the shared components. Pretty sure we can remove them later on once we upgrade the shared components repo.